pages/_app.tsx

```diff
@@ -44,7 +44,15 @@ export default function PlanetWeb({ Component, pageProps, err }: any) {
   }, []);
 
   React.useEffect(() => {
-    TagManager.initialize(tagManagerArgs);
+    console.log(process.env.NEXT_PUBLIC_GA_TRACKING_ID);
+    if (
+      process.env.NEXT_PUBLIC_GA_TRACKING_ID &&
+      (process.env.NEXT_PUBLIC_GA_TRACKING_ID !== undefined ||
+        process.env.NEXT_PUBLIC_GA_TRACKING_ID !== '' ||
+        process.env.NEXT_PUBLIC_GA_TRACKING_ID !== null)
```

Comment:
Out of curiosity: Isn't there some better way to check if value for existence then this for TypeScript or JavaScript in general?